stacks/api/function/queue/node/database.ts

```diff
@@ -0,0 +1,61 @@
+import {Database} from "@spica-server/function/queue/proto";
+import * as grpc from "grpc";
+
+export class DatabaseQueue {
+  private client: any;
```

Comment:
@thesayyn I think it would be better defining a more strict type for client.

Reply:
Current tools do not provide strict types for the service that is why I left this variable as any. 

Also, type information of the service changes regarding the proto files.